frontend/admin/package.json

```diff
@@ -52,13 +58,13 @@
     "@graphql-codegen/typescript-operations": "^2.5.10",
     "@graphql-codegen/typescript-urql": "^3.7.3",
     "@hydrogen-css/hydrogen": "2.0.0-beta.33",
-    "@storybook/addon-actions": "^6.5.13",
-    "@storybook/addon-essentials": "^6.5.15",
-    "@storybook/addon-links": "^6.5.15",
+    "@storybook/addon-actions": "6.5.14",
+    "@storybook/addon-essentials": "6.5.14",
+    "@storybook/addon-links": "6.5.14",
     "@storybook/addon-viewport": "^6.5.14",
```

Comment:
Does this need to version need to be static as well? Removing the '^'? Line 64
Just to stay in line with the other versions.

Reply:
oops, updated in [fix viewport version number](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5220/commits/9e987424a08ab608a6ce269700be97dbdaedbaf3)